### ISSUE TITLE
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

### ISSUE THREAD

**Issue by hrishidurg** [rancher-sandbox/rancher-desktop#4064]
### Actual Behavior

Am trying to build docker images for one my project work and seeing below errors ..Any WA for this ? 
I have started Rancher multiple times -but no luck .
==========
 ./build.sh
++ basename /Users/hdurg/sysint/streaming-transport/tools/synthetics
+ NAME=synthetics
++ git rev-parse --short=9 HEAD
+ VERSION=637e23d38
+ REPOSITORY=iad.ocir.io/sysintdev
+ TEAM=black]
+ docker build -t iad.ocir.io/sysintdev/black/synthetics:637e23d38 -f ./Dockerfile --build-arg BUILD_VERSION=637e23d38 .

Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

### Steps to Reproduce

./build.sh
++ basename /Users/hdurg/sysint/streaming-transport/tools/synthetics
+ NAME=synthetics
++ git rev-parse --short=9 HEAD
+ VERSION=637e23d38
+ REPOSITORY=iad.ocir.io/sysintdev
+ TEAM=black]
+ docker build -t iad.ocir.io/sysintdev/black/synthetics:637e23d38 -f ./Dockerfile --build-arg BUILD_VERSION=637e23d38 .

Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

### Result

./build.sh
++ basename /Users/hdurg/sysint/streaming-transport/tools/synthetics
+ NAME=synthetics
++ git rev-parse --short=9 HEAD
+ VERSION=637e23d38
+ REPOSITORY=iad.ocir.io/sysintdev
+ TEAM=black]
+ docker build -t iad.ocir.io/sysintdev/black/synthetics:637e23d38 -f ./Dockerfile --build-arg BUILD_VERSION=637e23d38 .

Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

### Expected Behavior

./build.sh
++ basename /Users/hdurg/sysint/streaming-transport/tools/synthetics
+ NAME=synthetics
++ git rev-parse --short=9 HEAD
+ VERSION=637e23d38
+ REPOSITORY=iad.ocir.io/sysintdev
+ TEAM=black]
+ docker build -t iad.ocir.io/sysintdev/black/synthetics:637e23d38 -f ./Dockerfile --build-arg BUILD_VERSION=637e23d38 .

Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

### Rancher Desktop Version

Version 1.7.0 (1.7.0)

### Rancher Desktop K8s Version

v4.5.4

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

Darwin hdurg-mac 21.4.0 Darwin Kernel Version 21.4.0: Mon Feb 21 20:35:58 PST 2022; root:xnu-8020.101.4~2/RELEASE_ARM64_T6000 arm64

### What CPU architecture are you using?

arm64 (Apple Silicon)

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by gunamata** [comment]
@hrishidurg , For some reason the docker daemon doesn't seem to be running on your machine. Do you have `dockerd(moby)` selected as the container engine in the `Preference > Container Engine` dialog menu?

**Comment by SiddharthChoudhary** [comment]
I am seeing the same issue. And yes @hrishidurg mine is dockerd(moby) selected.

**Comment by gunamata** [comment]
Do you have any other container management apps (ex: Docker Desktop, colima etc) running in parallel to Rancher Desktop? If yes, Please stop those apps and try launching Rancher Desktop again.

**Comment by hrishidurg** [comment]
@gunamata 
Yes , thats the first I did that (selected dockerd(moby) ) I fised this issue after several reinstall (dnt know exact RCA for fix) .

BUt when try to do build images using docker am seeing below error .

docker build -t iad.ocir.io/sysintdev/black/synthetics:ac6050d8f -f ./Dockerfile --build-arg BUILD_VERSION=ac6050d8f .docker push ${REPOSITORY}/${TEAM}/${NAME}:${VERSION}
======
no matching manifest for linux/arm64/v8 in the manifest list entries

Is there any suggestion or WA for the same .
Thanks

**Comment by gunamata** [comment]
> no matching manifest for linux/arm64/v8 in the manifest list entries

It looks like your Dockerfile pulls image(s) not built for your platform. Does `docker build` work for you for a different/simpler Dockerfile?

**Comment by gunamata** [comment]
Closing the issue as the original issue with docker not running seems to have been resolved after restarts. 
@hrishidurg , Please file a new issue if you see that it's not a platform issue and even a simpler Dockerfile doesn't work.
